### PR TITLE
Lighter initial loading and fixes 

### PR DESCRIPTION
**Pull request by frankrousseau**
This PR includes several fixes and a lighter inital loading:

* Loads events in a timeframe of 3 months before/after today.
* Make sure that non loaded events are loaded when needed (when browsing to unloaded month).
* Make calendar renaming operation lighter
* Make calendar deletion operation lighter
* Fix date locales in invitation emails
* Remove some Express warnings

The main goal of this PR is to avoid situation where the browser freezes (at least for a short time) and make initial loading faster.

### CODE REVIEW

File: client/app/router.coffee [hunk 1]
```diff
@@ -37,21 +46,28 @@ module.exports = class Router extends Backbone.Router
         else
             super route, options
 
+
     month: (year, month) ->
         if year?
-            @displayCalendar 'month', year, month, 1
+            monthToLoad = moment("#{year}/#{month}", "YYYY/M")
```
Comment (aenario) [inline client/app/router.coffee]: This action is not instantaneous anymore, may be it should have a spinner ?

**Comment by aenario** [comment]
Ok, except for my spinner comment above (wich can be added later). i have no issue with the changes or code. 

However Travis is unhappy as your test/event_model_test.coffee does not create the event/all request it uses. Do you have time to fix or should I ?

**Comment by frankrousseau** [comment]
@aenario I will handle all the comments. Thank you for the review.

**Comment by aenario** [comment]
I am closing in favor of #505 . Tests have been fixed. I will include the spinner in this sprint.

**Pull request closed without merging**